### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Generated by https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`

### CODE REVIEW

File: session.h [hunk 1]
```diff
@@ -23,7 +23,7 @@ struct session {
 struct session *session_new();
 void session_free(struct session *session);
 bool session_is_valid(struct session *session);
-struct session *sesssion_load(unsigned const char key[KDF_HASH_LEN]);
+struct session *session_load(unsigned const char key[KDF_HASH_LEN]);
```
Comment (jsoref) [inline session.h]: this is an api change...

File: xml.c [hunk 2]
```diff
@@ -252,7 +252,7 @@ xml_parse_share_user(xmlDoc *doc, xmlNode *item, struct share_user *user)
 			continue;
 		if (xml_parse_bool(doc, child, "group", &user->is_group))
 			continue;
-		if (xml_parse_bool(doc, child, "outsideenterpise", &user->outside_enterprise))
+		if (xml_parse_bool(doc, child, "outsideenterprise", &user->outside_enterprise))
```
Comment (jsoref) [inline xml.c]: this would be an api-like change. if the misspelling is actually intentional, it should be flagged.

**Comment by bcopeland** [comment]
Can you resend all of these with Signed-off-by?  Please see CONTRIBUTING.

**Comment by jsoref** [comment]
@bcopeland done

File: xml.c (hunk 2 above)
Comment (bcopeland) [reply xml.c]: This and other api-like changes are fine - this was a bug here.

**Pull request merged**